html_css/v2/main-course/positioning.md

```diff
@@ -0,0 +1,38 @@
+### Positioning
+
+You have learned various ways how to position elements on the screen and even though the methods you know so far are sufficient in most cases, these CSS rules are still very powerful and will certainly help you step up your CSS game!
```

Comment:
I think people are going to be confused about what we actually mean by "position" here because it _sounds_ like something they already know how to do (move stuff around a page with margin/padding/flexbox etc)... I think we could clear this up with how it's described in the introduction.... maybe something like this:

By now you have had quite a bit of practice moving elements around the screen using things like margin, padding, flexbox and grid. These techniques have all relied on CSS's default "positioning-mode".  This default positioning-mode is intuitive, and you'll continue using it for almost all of your layout needs, _however_, there are other methods at your disposal that can be very useful in some situations.